releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h

```diff
@@ -259,6 +259,28 @@ typedef struct FfiBoxedSlice {
   uintptr_t len;
 } FfiBoxedSlice;
 
+/**
+ * A struct that contains a pointer to, and length information for, a heap-allocated
+ * slice of `[u8; 32]` arrays.
+ *
+ * # Safety
+ *
+ * - `ptr` must be non-null and must be valid for reads for `len * mem::size_of::<[u8; 32]>()`
+ *   many bytes, and it must be properly aligned. This means in particular:
```

Comment:
As I read [the relevant Rust documentation](https://doc.rust-lang.org/reference/type-layout.html#array-layout), the alignment of `[u8; 32]`, or a slice of it, is guaranteed to be 1 (and, by applying the "`[T; n]` has the same alignment as `T`" rule twice, this is also true of `[[u8; 32]; len]` or a slice of it). Therefore "must be properly aligned" is vacuous here.